app/views/api/users/verify_pin.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.valid @valid
```

Comment:
`valid` is a not a semantically correct key - maybe `pin` (=valid/invalid/not_set)?